client/src/components/Framework/css/Cards.css

```diff
@@ -1,14 +1,43 @@
 .scoreboard {
 	display: flex;
-	width: 225px;
-	height: 350px;
-	background-color: #6b6b6b;
-	border-radius: 15px;
+	flex-direction: column;
+	width: 250px;
+	height: auto;
+	background-color: #707070d2;
+	border-radius: 10px;
 	box-sizing: border-box;
-	padding: 20px 30px 10px 30px;
+	padding: 5px;
 	line-height: 25px;
 }
 
+.sb-body {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	height: 100%;
+	border-radius: 0 0 7px 7px;
+	box-sizing: border-box;
+	padding: 10px 20px 10px 20px;
+	background-color: #202020;
+}
+
+.sb-header {
+	background-color: #ffffff;
+	width: 100%;
+	height: 40px;
+	text-align: center;
+	box-sizing: border-box;
+	padding: 10px;
+	border-radius: 5px 5px 0 0;
+	font-weight: 600;
+}
+
+.sb-player {
+	font-weight: 600;
+	color: #fafafa;
```

Comment:
if we're not using SASS for now, maybe put a comment next to each colour? so like `color: #fafafa; //dark grey`

Reply:
not everyone has an innate knowledge of hex codes haha